src/docs/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -19,11 +20,6 @@ As of Flutter 2.5, we support the following platforms:
 |Web     | Edge 1.2.0 & above           | All     |
 |Windows | Windows 7 & above            | All     |
 
-
-Note that while macOS and Linux, are in the stable channel to enable you to
```

Comment:
I'm curious why this block has been deleted. The desktop beta snapshot in the stable channel is still beta at this point.

Reply:
Because Ian removed it from *his* PR. I can update it, though.